style.css

```diff
@@ -0,0 +1,422 @@
+    align-self: center;
+    border-radius: 50px;
+    padding: 12px 16px;
+    background-color: #8ecdf8;
+}
+
+.new-tweet-button > p {
+    color: white;
+    font-weight: bold;
+}
+
+.tweet-text-input:focus {
+    outline: none;
+}
+
+/* Twitter Feed */
+
+.twitter-feed {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(3, min-content)
+}
+
+.a-tweet {
+    display: grid;
+    grid-template-columns: 50px 1fr;
+    grid-template-rows: 25px min-content 50px;
+    column-gap: 12px;
+    padding: 12px 12px 0px 12px;
+    grid-template-areas:
+    "pic tweet-top"
+    "pic tweet-mid"
+    "pic tweet-bottom";
+    border-bottom: solid 1px lightgray;
+}
+
+.a-tweet-profile-pic {
+    grid-area: pic;
+    border-radius: 50%;
+    overflow: hidden;
+    height: 50px;
+}
+
+.a-tweet-profile-pic > img {
+    height: 50px;
+}
+
+.a-tweet-details {
+    grid-area: tweet-top;
+    display: grid;
+    grid-template-columns: repeat(4, auto) 1fr;
+    grid-template-rows: auto;
+    grid-gap: 5px;
+    font-size: 14px;
+}
+
+.a-tweet-body {
+    grid-area: tweet-mid;
+}
+
+.a-tweet-dname {
+    font-weight: bold;
+}
+
+.a-tweet-icons {
+    grid-area: tweet-bottom;
+    display: grid;
+    grid-template-columns: repeat(5, auto);
+    align-items: center;
+    padding-top: 10px;
+}
+
+.a-tweet-icons img {
+    width: 40px;
+    border-radius: 20px;
+    padding: 10px;
+}
+
+.reply-icon:hover {
+    background-color: #e1eef6;
+}
+
+.retweet-icon:hover {
+    background-color: #def1eb;
+}
+
+.like-icon:hover {
+    background-color: #f7e0eb;
+}
+
+.share-icon:hover {
+    background-color: #e1eef6;
+}
+
+/* Right Column */
+
+.right-column {
+    grid-area: right;
+    display: grid;
+    gap: 10px;
+    grid-template-rows: 50px auto auto 1fr;
+    grid-template-areas:
+    "search"
+    "happen"
+    "follow";
+    padding: 5px 20px;
+}
+
+.search {
+    grid-area: search;
+    display: grid;
+    grid-template-columns: 20px 1fr;
+    /* border: solid 1px red; */
+    padding: 12px;
+    grid-gap: 10px;
+    background-color: lightgrey;
+    border-radius: 100px;
+}
+
+.search-bar {
+    width: 280px;
+    border: none;
+    background-color: lightgray;
+    font-size: 16px;
+}
+
+.search-bar:focus {
+    outline: none;
+}
+
+.whats-happening {
+    grid-area: happen;
+    padding: 10px;
+    margin-top: 20px;
+    display: grid;
+    width: 100%;
+    height: auto;
+    background-color: rgb(233, 233, 233);
+    border-radius: 15px;
+}
+
+.whats-happening > p {
+    padding-bottom: 10px;
+    border-bottom: solid 1px rgb(224, 224, 224);
+}
+
+.who-to-follow {
+    grid-area: follow;
+    padding: 10px;
+    display: grid;
+    width: 100%;
+    height: auto;
+    background-color: rgb(233, 233, 233);
+    border-radius: 15px;
+}
+
+.who-to-follow > p {
+    padding-bottom: 10px;
+    border-bottom: solid 1px rgb(224, 224, 224);
+}
+
+.right-column .title {
+    font-weight: bold;
+    font-size: 18px;
+}
```

Comment:
I will see if I can clone your work over the weekend for a better view. If I have any comments then, I shall drop you some. Well done!.